client/app/bundles/App/startup/registration.jsx

```diff
@@ -0,0 +1,22 @@
+import ReactOnRails from 'react-on-rails';
+import $ from 'jquery'
+
+import Router from '../components/Router';
+
+/* Utils */
+String.prototype.capitalize = function() {
```

Comment:
Note that manipulating the prototype is considered a bad and dangerous practice, it would probably be better to use `lodash`, something like

```
import { capitalize } from 'lodash';

... let string = capialize(string);
```